pages/common/snowsql.md

```diff
@@ -0,0 +1,24 @@
+# snowsql
+
+> SnowSQL command-line client for Snowflake's Data Cloud.
+> More information: <https://docs.snowflake.com/en/user-guide/snowsql.html>.
+
+- Connect to an instance at {{account}}.snowflakecomputing.com (password can be provided in prompt or configuration file):
```

Comment:
```suggestion
- Connect to a specific instance at account.snowflakecomputing.com (password can be provided in prompt or configuration file):
```

- This change is optional - no corresponding style guide requirement exist.
- The corresponding change should be applied to `.pl` translation if it's used here.
- We often prefer use `a specific` statement.
- There is no requirement to enclose referred to placeholders in braces in descriptions.